docs/GettingStarted.md

```diff
@@ -39,6 +39,16 @@ $ curl -L https://github.com/mozilla/geckodriver/releases/download/v0.21.0/gecko
 
 Note: Other geckodriver releases are available [here](https://github.com/mozilla/geckodriver/releases). In order to automate other browser you need to run different drivers. You can find a list with all drivers in the [awesome-selenium](https://github.com/christian-bromann/awesome-selenium#driver) readme.
 
+Windows 64 bit (Powershell)
+
+```sh
+# Use geckodriver-v0.21.0-win32.zip for 32 bit Windows
+$url = "https://github.com/mozilla/geckodriver/releases/download/v0.21.0/geckodriver-v0.21.0-win64.zip"
+$output = "geckodriver.zip" # will drop into current directory unless defined otherwise
+
+Invoke-WebRequest -Uri $url -OutFile $output
```

Comment:
The windows example should be above the note in line 40, don't see a reason to separate it.

Reply:
maybe we should just use the geckodriver NPM package https://www.npmjs.com/package/geckodriver#use-it-globally

Reply:
Whoops yeah saw the change. Moved the Windows instructions before the note.

Hadn't used the geckodriver NPM package before but tried it out. I'm not opposed to doing that but did find the geckodriver on the Chocolatey package system. If you are not familiar just think homebrew but for Windows users and installed that without a hitch. 

Let me know if you'd me to pivot away from my simple and advanced instructions to something else! Thanks for the ping!